star-rx72n-firmware/lib/rx_hal/src/rspi.c

```
@@ -531,3 +558,476 @@ rx_err_t rspi_deinit(const uint8_t channel)
+  if (err != k_rx_ok) {
+    return err;
+  }
+
+  /* Configure CS GPIO pin */
+  err = internal_configure_cs_gpio(rx_port_from_pin(config->cs), rx_pin_from_pin(config->cs));
+  if (err != k_rx_ok) {
+    return err;
+  }
+
+  /* Get RSPI base */
+  volatile rx_rspi_regs_t* rspi = internal_get_rspi_base(channel);
```

Clang (14.0.6)

[warning] variable 'rspi' is not initialized

(cppcoreguidelines-init-variables)